install_config/install/disconnected_install.adoc

```diff
@@ -0,0 +1,550 @@
+----
+====
++
+changes to
++
+====
+----
+"spec": {
+  "dockerImageRepository": "172.30.69.44:5000/openshift/mongodb-26-rhel7",
+----
+====
++
+[NOTE]
+====
+The repository name (rhscl) was changed to openshift. You will need
+to ensure the change, regardless of whether the repository is rhscl, openshift3,
+or another directory. Every definition should have the following format:
+
+----
+<REGISTRY_IP>:5000/openshift/<IMAGENAME>
+----
+====
++
+Repeat this for every imagestream in the file. Ensure you use the correct IP
+address that you determined earlier. When you are finished, save and exit.
+Repeat the same process for the JBoss imagestreams in
+*_/usr/share/openshift/examples/xpaas-streams/jboss-image-streams.json_*.
+
+. Load the updated imagestream definitions:
++
+----
+# oc create -f /usr/share/openshift/examples/image-streams/image-streams-rhel7.json -n openshift
+# oc create -f /usr/share/openshift/examples/xpaas-streams/jboss-image-streams.json -n openshift
+----
+
+[[disconnected-load-the-docker-images]]
+=== Load the Docker Images
+
+At this point the system has been prepared for loading the Docker images.
+
+. Log in to the Docker registry using the token and registry service IP
+obtained earlier:
++
+----
+# docker login -u adminuser -e mailto:adminuser@abc.com \
+   -p $MYTOKEN $REGISTRY:5000
+----
+
+. Push the Docker images:
++
+----
+# docker push $REGISTRY:5000/openshift/amq-openshift
+# docker push $REGISTRY:5000/openshift/eap-openshift
+# docker push $REGISTRY:5000/openshift/tomcat7-openshift
+# docker push $REGISTRY:5000/openshift/tomcat8-openshift
+# docker push $REGISTRY:5000/openshift/mongodb-26-rhel7
+# docker push $REGISTRY:5000/openshift/mysql-56-rhel7
+# docker push $REGISTRY:5000/openshift/perl-520-rhel7
+# docker push $REGISTRY:5000/openshift/php-56-rhel7
+# docker push $REGISTRY:5000/openshift/postgresql-94-rhel7
+# docker push $REGISTRY:5000/openshift/python-27-rhel7
+# docker push $REGISTRY:5000/openshift/python-34-rhel7
+# docker push $REGISTRY:5000/openshift/ruby-22-rhel7
+# docker push $REGISTRY:5000/openshift/nodejs-010-rhel7 
+----
+
+. Verify the that all the imagestreams now have the tags populated:
++
+----
+# oc get imagestreams -n openshift
+----
++
+The output should look similar to the following output:
++
+====
+----
+NAME                                 DOCKER REPO                                                      TAGS                                     UPDATED
+jboss-amq-62                          registry.access.redhat.com/jboss-amq-6/amq62-openshift                         1.1,1.1-2,1.1-6 + 2 more...     2 weeks ago            
+...
+----
+====
+
+[[disconnected-install-a-router]]
+== Install a Router
+
+At this point the OpenShift environment is almost ready for use. It is likely
+that you will want to install and configure a router, documented here:
+
+https://docs.openshift.com/enterprise/3.1/install_config/install/deploy_router.html 
```

Comment:
Suggest using `link:` w/ that URL, w/ the link text "install and configure a router".